src/external/xgboost/CMakeLists.txt

```diff
@@ -4,11 +4,11 @@ make_library(xgboost
   SOURCES
     src/gbm/gbm.cpp
     src/tree/updater.cpp
+    src/io/dmlc_simple.cpp
     src/io/io.cpp
     subtree/rabit/src/engine_empty.cc
   REQUIRES
     libjson
-    dmlc
```

Comment:
If xgboost doesn't need dmlc, do we still need it at all?

Reply:
I can't find where it's building the `dmlc` library - nothing else in the CMakeLists seems to produce it. But if you can find it let's remove it.